### DISCUSSION TITLE
Regarding octernship

### DISCUSSION THREAD

**Discussion by shashwatw** [SecureStackCo/app.cheapcryptobank.com#1] [Q&A]
I applied for the task by SecureStack but am not getting perfect idea of what has to be done for the task.
The GitHub repo doesn't contain any app.js file in it and just consists of config files. Being a beginner can anyone please help me that what exactly should be done...?

**Comment by prashant235913** [comment]
@shashwatwankhedekar Its a web app so and I think This is the link:- [https://app.cheapcryptobank.com/](url)
